### ISSUE TITLE
Monorepo bug with language server

### ISSUE THREAD

**Issue by TheComputerM** [sveltejs/language-tools#520]
**Describe the bug**
Nested svelte.config.js does not work

**To Reproduce**
https://github.com/TheComputerM/lang-server-bug

**Expected behavior**
Filepaths should be relative to nested svelte.config.js

**Screenshots**
![image](https://user-images.githubusercontent.com/24741538/92137355-a7d4ce00-ee2a-11ea-8437-8a2bdacc37aa.png)

**System (please complete the following information):**
 - OS: Windows
 - IDE: VSCode
 - Plugin/Package: Svelte for VSCode

**Comment by dummdidumm** [comment]
Doing 
```js
scss: {
      includePaths: ['packages/package1/src', 'packages/package1/theme'],
    },
```
works for the language-server, but then does no longer work for the build. So a workaround in the meantime would be to split up the definition into one for build and one for the ls.

**Comment by TheComputerM** [comment]
I do not know anything about vs code extensions, but [this](https://github.com/microsoft/vscode/wiki/Adopting-Multi-Root-Workspace-APIs#eliminating-rootpath) might help.

**Comment by TheComputerM** [comment]
Found a fix:
- Make a VS Code multi-root workspace
- Make folder with svelte.config.js the first in the workspace array because [this](https://code.visualstudio.com/docs/editor/multi-root-workspaces#_extensions).
![image](https://user-images.githubusercontent.com/24741538/92203322-7c8bc680-ee9e-11ea-900f-fd0a76e730a7.png)

**Comment by dummdidumm** [comment]
A workaround which will work for both build and language server:

```js
scss: {
      includePaths: [path.join(__dirname, 'src', path.join(__dirname, 'theme'],
},
```

**Comment by TheComputerM** [comment]
Ok then I will mark this issue as resolved, consider adding [multi root workspace api](https://github.com/microsoft/vscode/wiki/Adopting-Multi-Root-Workspace-APIs#eliminating-rootpath) in the future if possible.

**Comment by dummdidumm** [comment]
If I understand the multi root workspace documentation correctly, it would only help solve your problem if we would start one language server per workspace root, which - according to the documentation - is not recommended. Basic support for multiple folders was implemented here btw #480 .
